storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -275,6 +276,37 @@ func (s *commitLogSource) shouldEncodeSeries(
 	return ranges.Overlaps(blockRange)
 }
 
+func (s *commitLogSource) shouldIncludeInIndex(
```

Comment:
nit: Maybe `shouldIndexSeries` is more similar to the other method?

Reply:
Yeah I'm planning on renaming the other method. I think its a bad name because it evaluates its on a datapoint-per-datapoint basis

Reply:
Just didn't wanna do it in this P.R

Reply:
No problems.